### ISSUE TITLE
Static assets

### ISSUE THREAD

**Issue by ferllings** [aklinker1/vite-plugin-web-extension#167]
Hello,
More a question than a bug.

I have some 3rd party libs, already minified, that I would like to include in my manifest:
```
          "js": [
              "external/lib.min.js"
          ],
```

How can include it, without processing/building it?

Thanks

**Comment by aklinker1** [comment]
Put it in [Vite's public directory](https://vitejs.dev/guide/assets.html#the-public-directory).

**Comment by ferllings** [comment]
Yes, but I can not longer keep the record in the manifest: 
The file is in /public/lib/leader-line.min.js, 
```
✓ 0 modules transformed.
Could not resolve entry module "src/lib/leader-line.min.js".
✓ 0 modules transformed.
[web-extension:manifest] Could not resolve entry module "src/lib/leader-line.min.js".
```

**Comment by ferllings** [comment]
Nevermind, I believe I found my mistake:
It's working now
```
            "js": [
                "public/lib/leader-line.min.js"
            ],
```

**Comment by ferllings** [comment]
Looks like I spook too soon:

I put my lib into
src/public/leader-line.min.js

Then in manifest 
```
            "js": [
                "public/leader-line.min.js"
            ],
```

Vite put a unmodified copy on "dist/leader-line.min.js"
and second but modified copy on "dist/public/leader-line.min.js"

If switch manifest.json, in order to use the "dist/leader-line.min.js" unmodified copy
```
            "js": [
                "leader-line.min.js"
            ],
```
It throws
Could not resolve entry module "src/leader-line.min.js"

**Comment by aklinker1** [comment]
@ferllings I've added a new feature to address this, could you try it out? 

Install version `4.1.0-alpha1` (the next tag on NPM), and add the `"public:"` prefix to your content script path.

```
"js": [
  "public:leader-line.min.js"
],
```

Make sure the file is in `public/leader-line.min.js`, and it should work for your.

**Comment by ferllings** [comment]
I'm afraid it still not working:
Could not resolve entry module "public:leader-line.min.js"

I modified the manifest file:
<img width="428" alt="Screenshot 2023-12-19 at 10 25 18" src="https://github.com/aklinker1/vite-plugin-web-extension/assets/2118130/b45812d1-add5-4894-b9f5-02e09d2c877b">

The installed node_module version is correct:
<img width="681" alt="Screenshot 2023-12-19 at 10 21 52" src="https://github.com/aklinker1/vite-plugin-web-extension/assets/2118130/1a6fd718-cc5c-443a-9e9c-fb5644e2ebc9">

**Comment by aklinker1** [comment]
Interesting. Could you upload a zip of a minimal reproduction? The above was working for me in my test case.

**Comment by ferllings** [comment]
Here you go
[test-repo.zip](https://github.com/aklinker1/vite-plugin-web-extension/files/13724745/test-repo.zip)

**Comment by aklinker1** [comment]
Fix released in [`v4.1.0`](https://github.com/aklinker1/vite-plugin-web-extension/releases/tag/v4.1.0)

**Comment by ferllings** [comment]
Thanks. I confirm it is working.
